IndicoToolkit/Types/Position.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+
+using IndicoToolkit.Exception;
+
+namespace IndicoToolkit.Types
+{
+    public class Position
+    {
+        public int top { get; set; }
+        public int bottom { get; set; }
+        public int left { get; set; }
+        public int right { get; set; }
+        public float bbTop {get; set;}
+        public float bbBot {get; set;}
+        public float bbLeft {get; set;}
+        public float bbRight {get; set;}
+        public int pageNum {get; set;}
+
+        public Position(
+            int top,
+            int bottom,
+            int left,
+            int right,
+            float bbTop,
+            float bbBot,
+            float bbLeft,
+            float bbRight,
+            int pageNum
+        )
+        {
+            this.top = top;
+            this.bottom = bottom;
+            this.left = left;
+            this.right = right;
+            this.bbTop = bbTop;
+            this.bbBot = bbBot;
+            this.bbLeft = bbLeft;
+            this.bbRight = bbRight;
+            this.pageNum = pageNum;
+        }
+
+        public Tuple<float, float> getCorner(string leftOrRight, string topOrBot)
```

Comment:
is this easier than 4 methods, "getBottomLeftCorner"?

Just thinking aloud... no need to change

Reply:
Yeah I opted for a more compact solution, but I'll defer to you guys on what you think is easier to use. 

Was also thinking along the lines of if a user makes a mistake or wants to get multiple corners, can always just change kwargs instead of writing new function name, etc.